docs/using.md

```diff
@@ -34,67 +35,76 @@ This section contains information about basic virtual machine and OpenShift mana
 
-Minishift allows you to specify command line flags you commonly use via environment variables.
-To do so, apply the following rules to the flag you want to set via an environment variable.
+Minishift allows you to specify command line flags you commonly use through environment variables.
+To do so, apply the following rules to the flag you want to set as an environment variable.
 
-* Apply `MINISHIFT_` as a prefix to your environment variable, for example the `vm-driver` flag
-  of the [start](./minishift_start.md) command becomes `MINISHIFT_vm-driver`.
-* Uppercase the flag, `MINISHIFT_vm-driver` becomes `MINISHIFT_VM-DRIVER`.
-* Last but not least, replace `-` with `_`, `MINISHIFT_VM-DRIVER` becomes `MINISHIFT_VM_DRIVER`
+1. Apply `MINISHIFT_` as a prefix to the flag you want to set as an environment variable. For example, the `vm-driver` flag
+  of the [`minishift start`](./minishift_start.md) command becomes `MINISHIFT_vm-driver`.
+1. Use uppercase for the flag, so `MINISHIFT_vm-driver` in the above example becomes `MINISHIFT_VM-DRIVER`.
+1. Finally, replace `-` with `_`, so `MINISHIFT_VM-DRIVER` becomes `MINISHIFT_VM_DRIVER`.
 
-Another common example might be the URL of the ISO to be used. Usually you specify it via
-`iso-url` of the [start](./minishift_start.md) command. Applying the rules from above, you can
-also specify this URL by setting the environment variable `MINISHIFT_ISO_URL`.
+Another common example is the URL of the ISO to be used. Usually you specify it with the
+`iso-url` flag of the [`minishift start`](./minishift_start.md) command. Applying the above rules, you can
+also specify this URL by setting the environment variable as `MINISHIFT_ISO_URL`.
 
-**Note:** There is also the `MINISHIFT_HOME` environment variable. Per default Minishift places all
-its runtime state into `~/.minishift`. Using `MINISHIFT_HOME`, you can choose a different directory
-as Minishift's home directory. This is currently experimental and semantics might change in
+**Note:** You can also use the `MINISHIFT_HOME` environment variable, to choose a different home directory for Minishift. Per default, Minishift places all
+its runtime state into `~/.minishift`.
+ This is currently experimental and semantics might change in
 future releases.
 
 <a name="persistent-configuration"></a>
-## Persistent configuration
-
-Minishift also maintains a configuration file (`$MINISHIFT_HOME/config/config.json`) which can be
-used to set commonly used command-line flags persistently. For example `cpus`, `memory` or `vm-driver`.
-For a full set of supported configuration options refer to the synopsis of the
-[config](./minishift_config.md) sub-command.
+### Persistent configuration
 
-<a name="configuration-options-precedence"></a>
-### Configuration options precedence
+Using persistent configuration allows you to control Minishift's behavior without specifying actual command line flags, similar to the way you use [environment variables](#environment-variables).
```

Comment:
This is probably the point to make the distinction that env variables can be used for any odd options, whereas persistent configuration can only be applied to the options listed in config command synopsis (see comment above)